templates/velero.yaml

```diff
@@ -7,391 +63,42 @@ metadata:
-        creationTimestamp: null
-        labels:
-          component: velero
-        name: velero
-        namespace: velero
-      spec:
-        selector:
-          matchLabels:
-            deploy: velero
-        strategy: {}
-        template:
-          metadata:
-            annotations:
-              prometheus.io/path: /metrics
-              prometheus.io/port: "8085"
-              prometheus.io/scrape: "true"
-            creationTimestamp: null
-            labels:
-              component: velero
-              deploy: velero
-          spec:
-            containers:
-            - args:
-              - server
-              command:
-              - /velero
-              env:
-              - name: VELERO_SCRATCH_DIR
-                value: /scratch
-              - name: GOOGLE_APPLICATION_CREDENTIALS
-                value: /credentials/cloud
-              - name: AWS_SHARED_CREDENTIALS_FILE
-                value: /credentials/cloud
-              - name: AZURE_CREDENTIALS_FILE
-                value: /credentials/cloud
-              image: gcr.io/heptio-images/velero:v1.0.0
-              imagePullPolicy: IfNotPresent
-              name: velero
-              ports:
-              - containerPort: 8085
-                name: metrics
-              resources: {}
-              volumeMounts:
-              - mountPath: /plugins
-                name: plugins
-              - mountPath: /scratch
-                name: scratch
-              - mountPath: /credentials
-                name: minio-credentials
-            restartPolicy: Always
-            serviceAccountName: velero
-            volumes:
-            - emptyDir: {}
-              name: plugins
-            - emptyDir: {}
-              name: scratch
-            - name: minio-credentials
-              secret:
-                secretName: minio-credentials
-    ---
-    apiVersion: velero.io/v1
-    kind: Schedule
-    metadata:
-      name: daily-backup
-      namespace: velero
-    spec:
-      schedule: 0 1 * * *
-      template:
-        includedNamespaces:
-        - '*'
-    ---
-    apiVersion: batch/v1
-    kind: Job
-    metadata:
-      namespace: velero
-      name: minio-setup
-      labels:
-        component: minio
-    spec:
-      template:
-        metadata:
-          name: minio-setup
-        spec:
-          restartPolicy: OnFailure
-          volumes:
-          - name: config
-            emptyDir: {}
-          containers:
-          - name: mc
-            image: minio/mc:latest
-            imagePullPolicy: IfNotPresent
-            command:
-            - /bin/sh
-            - -c
-            - "mc --config-dir=/config config host add velero http://minio:9000 minio CHANGEME && mc --config-dir=/config mb -p velero/velero"
-            volumeMounts:
-            - name: config
-              mountPath: "/config"
+  chartReference:
+    chart: stable/velero
+    version: 2.1.1
+    values: |
+      ---
+      configuration:
```

Comment:
Can we enable metrics exporting? There is a `metrics.enabled` config field that is defaulted to `false`

Reply:
Sure, sounds good :+1:

Reply:
feb57b8bc82d7b8007d448069af3e897cb3c10f6 takes care of enabling metrics, and enabling the serviceMonitor for the Prometheus operator. Let me know if there's anything else you'd like tweaked before we merge @gracedo :+1: